lib/backpex/html/resource.ex

```diff
@@ -175,7 +175,13 @@ defmodule Backpex.HTML.Resource do
       |> assign(:field, field)
       |> assign(:field_options, field_options)
       |> assign(:type, :form)
-      |> assign(:readonly, Backpex.Field.readonly?(field_options, assigns))
+
+    assigns =
+      if assigns[:readonly] == true do
+        assigns
+      else
+        assign(assigns, :readonly, Backpex.Field.readonly?(field_options, assigns))
+      end
```

Comment:
For which cases do we need this?

And can we simplify the if-condition?


```suggestion
    assigns =
      if assigns[:readonly] do
        assigns
      else
        assign(assigns, :readonly, Backpex.Field.readonly?(field_options, assigns))
      end
```